Alguns dados interessantes

felquis opened the pull request:
Informações que acho pertinente para começar o README da organização, que seria um resumo do Ionic Brazil, mas mais da nossa missão, organização, e participação da galera.

O repositório do site, usamos como um resumo mais fácil de digerir de toda a organização, porém tudo lincando para cá, contato, como entrar na organização etc...

O que vocês acham?

felquis committed:
Alguns dados interessantes

Informações que acho pertinente para começar o README da organização, que seria um resumo do Ionic Brazil, mas mais da nossa missão, organização, e participação da galera.

O repositório do site, usamos como um resumo mais fácil de digerir de toda a organização, porém tudo lincando para cá, contato, como entrar na organização etc...

felquis commented:
cc @IonicBrazil/moderadores

diegosorrilha commented:
Acho legal @felquis ... assim a informação fica mais centralizada.

Estou correndo essa semana.. mas a partir da semana pretendo colaborar mais ;)

breakzplatform commented:
+1 @juarezpaf @r4bugento

iamfelipemattos commented:
perfeito felquis!
+1 @juarezpaf

felquis commented:
Bacana galera, podem comentar na linha onde vocês vêem uma alteração/correção por favor?

felquis committed:
Melhorias sugeridas pelo Juarez

https://github.com/IonicBrazil/organizacao/commit/58915d3fcd438ed849d3ec93b01c0106d30f892e

felquis committed:
Mais melhorias no texto

https://github.com/IonicBrazil/organizacao/commit/58915d3fcd438ed849d3ec93b01c0106d30f892e

felquis commented:
@juarezpaf e agora como ficamos? :D

juarezpaf commented:
:rocket:
